crates/ruff/src/rules/flake8_todo/snapshots/ruff__rules__flake8_todo__tests__missing-link-in-todo_TDO003.py.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: crates/ruff/src/rules/flake8_todo/mod.rs
+---
+TDO003.py:9:3: TDO003 Missing issue link following TODO
+   |
+ 9 | # TDO003 - errors
+10 | # TODO: this comment has no
+   |   ^^^^ TDO003
```

Comment:
Is the link expected to always appear at the end or on the next line?

Reply:
On the next line. I'm thinking of flagging the line with the `TODO` and saying along the lines of "the next line should have an issue link". I think it would break if we tried throwing the diagnostic on the line after the `TODO` in the case where the `TODO` is the last line of the file.